lib/core/form/components/inplace-mat-form-field-wrapper/inplace-mat-form-field-wrapper.component.html

```diff
@@ -0,0 +1,3 @@
+<div [ngClass]="{ 'adf-inplace-mat-form-field-wrapper': true, 'adf-inplace-mat-form-field-wrapper-error': isInvalid }">
```

Comment:
If adf-inplace-mat-form-field-wrappe is always there as style isn't it possible to add it as simple class?

Reply:
you mean something like below?

```
[ngClass]="[ 
  'adf-inplace-mat-form-field-wrapper',
   isInvalid ? 'adf-inplace-mat-form-field-wrapper-error' : ''
] 
```